package.json

```diff
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "test": "jest --watch"
```

Comment:
Uh oh! I think  `--watch` is causing the GH action to stall

Reply:
The test job was actually running for 1 hour. I had to cancel it!

Reply:
Oh crap! Fixed.